storage/scylla_api.c

```diff
@@ -403,6 +400,73 @@ static status_t create_edge_table(CassSession* session) {
 exit:
   return ret;
 }
+
+status_t create_identity_table(CassSession* session, bool is_need_drop) {
+  status_t ret = SC_OK;
+  if (is_need_drop) {
+    if (execute_query(session, "DROP TABLE IF EXISTS identity;") != CASS_OK) {
+      ta_log_error("drop identity table fail\n");
+      ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
+      goto exit;
+    }
+  }
+  if (execute_query(session,
+                    "CREATE TABLE IF NOT EXISTS identity("
+                    "code uuid, hash blob, status tinyint, PRIMARY KEY (code));") != CASS_OK) {
+    ta_log_error("create identity table fail\n");
+    ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
+    goto exit;
+  }
+  if (execute_query(session, "CREATE INDEX IF NOT EXISTS ON identity(status);") != CASS_OK) {
+    ta_log_error("create identity table index fail\n");
+    ret = SC_STORAGE_CASSANDRA_QUREY_FAIL;
+    goto exit;
+  }
+exit:
+  return ret;
+}
+status_t init_identity(CassCluster** cluster, CassSession* session, char* hosts, bool is_need_drop,
+                       const char* keyspace_name) {
+  if (hosts == NULL) {
```

Comment:
Do we need to examine `cluster` and `session`

Reply:
I am refactoring about the initialization.